auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+        description: Returns scheduled changes for rules.
+      parameters:
+        - $ref: '#/parameters/allSCParam'
+      responses:
+        '200':
+          description: Returns JSON object of list of all scheduled changes for rules plus total count
+          schema:
+            type: object
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Rule
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCUptake'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/RulesBase'
+                        - $ref: '#/definitions/RulesSCNullable'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                      required:
+                        - alias
+                        - backgroundRate
+                        - buildID
+                        - buildTarget
+                        - change_type
+                        - channel
+                        - comment
+                        - data_version
+                        - distVersion
+                        - distribution
+                        - fallbackMapping
+                        - headerArchitecture
+                        - instructionSet
+                        - locale
+                        - mapping
+                        - memory
+                        - osVersion
+                        - priority
+                        - product
+                        - rule_id
+                        - telemetry_channel
+                        - telemetry_product
+                        - telemetry_uptake
+                        - update_type
+                        - version
+                        - when
+    post:
+      summary: create scheduled change for rules.
+      description: >
+        Create scheduled change for a rule(existing or new). [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id34)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_rules_post
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: "create scheduled change for rule."
+      parameters:
+        - name: change_type
```

Comment:
because this one is the query (optional) parameter passed for the /sc/rules post operation and already handled [here](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/views/rules.py#L296).